mitodl/micromasters
# Removed x's from text fields

#### What are the relevant tickets?
Fixes #612 

#### What's this PR do?
Removes the X's on IE11

#### How should this be manually tested?
Load an IE11 VM, go to /users/ and confirm that this X doesn't appear when you edit a text field.

#### Any background context you want to provide?
http://stackoverflow.com/questions/14007655/remove-ie10s-clear-field-x-button-on-certain-inputs
